additional-databases/neptune/README.md

```diff
@@ -0,0 +1,38 @@
+## Connecting a local graph-notebook to Amazon Neptune (first-time setup)
+When using graph-notebook locally to connect to an Amazon Neptune database for the first time, there are a couple of additional steps. This section assumes that you've already installed & configured [graph-notebook](#installation) locally.  Please note that this wiki is not an official recommendation on network setups as there are many ways to connect to Amazon Neptune from outside of the VPC, such as setting up a load balancer or VPC peering.
+
+Amazon Neptune DB clusters can only be created in an Amazon Virtual Private Cloud (VPC).  One way to connect to Amazon Neptune from outside of the VPC is to set up an Amazon EC2 instance as a proxy server within the same VPC. With this approach, you will also want to set up an SSH tunnel to securely forward traffic to the VPC. 
+
+### Part 1: Set up a EC2 proxy server.
+Launch an [Amazon EC2](https://aws.amazon.com/ec2/) instance located in the same region as your Neptune cluster. In terms of configuration, a standard Amazon Linux AMI can be used.  Since this is a proxy server, you can choose the lowest resource settings.  
+
+Make sure the EC2 instance is in the same VPC group as your Neptune cluster. To find the VPC group for your Neptune cluster, check the console under [Neptune](https://console.aws.amazon.com/neptune/home) > Subnet groups. The instance's security group needs to be able to send and receive on port 22 for SSH and port 8182 for Neptune.  See below for an example security group setup.  
+
+![Sample EC2 Inbound Rules](/././images/sample-ec2rules.png)
```

Comment:
I do not think this image url will work, maybe something like:

\!\[Sample EC2 Inbound Rules](https://github.com/aws/graph-notebook/blob/main/images/sample-ec2rules.png)